code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -4,13 +4,38 @@
 	icon = 'icons/obj/crates.dmi'
 	icon_state = "densecrate"
 	density = 1
+	var/obj/item/weapon/paper/manifest/manifest
+

```

Comment:
The fact that largecrate is not a crate prevents the use of the tear_manifest() unless I drag tear_manifest() to structures but then you could tear manifests from every structure (maybe in the future cargo could order complete machines).